word_embedding/train_v2.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import paddle.v2 as paddle
+from network_conf import network_conf
+import gzip
+
+
+def main():
+    paddle.init(use_gpu=False, trainer_count=1)
+    word_dict = paddle.dataset.imikolov.build_dict(typo_freq=2)
```

Comment:
It is better to rename `type_freq` into `min_word_freq` because the word `typo` is ambiguous.